GitWorkflow/GitWorkflow/Player.cpp

```diff
@@ -0,0 +1,77 @@
+#include "Player.hpp"
+
+Player::Player(const sf::Vector2f& position, const sf::Vector2f& size, const sf::Color& color)
+	: Drawable(), m_model(size)
+{
+	m_model.setFillColor(color);
+	m_model.setPosition(position);
+}
+
+void Player::move(const sf::Vector2f& offset)
+{
+	m_model.move(offset);
+}
+
+void Player::setPosition(const sf::Vector2f& position)
+{
+	m_model.setPosition(position);
+}
+
+const sf::Vector2f& Player::getPosition() const
+{
+	return m_model.getPosition();
+}
+
+void Player::setVelocity(const sf::Vector2f& velocity)
+{
+	m_velocity = velocity;
+}
+
+const sf::Vector2f& Player::getVelocity() const
+{
+	return m_velocity;
+}
+
+const sf::FloatRect Player::getGlobalBounds() const
+{
+	return m_model.getGlobalBounds();
+}
+
+void Player::collide(const Platform& platform)
+{
+	auto intersection = util::intersection(getGlobalBounds(), platform.getGlobalBounds());
```

Comment:
Why no const here? Reference is not needed, though

Reply:
Fixed.